### ISSUE TITLE
Loadup says SUCCESS when it actually failed

### ISSUE THREAD

**Issue by rmkaplan** [Interlisp/medley#1724]
**Describe the bug**
If a Lisp error happens during the loadup-full.sh script, a full.sysout is created in the loadups directory and the terminal window reports SUCCESS.

But in fact, the loadup failed, files after the error are not included.  The sysout is smaller than it should be, as shown in this screen shot:
![Screenshot 2024-05-20 at 9 44 23 PM](https://github.com/Interlisp/medley/assets/69548581/3d4b195c-8ceb-49b9-ba74-33b2bed1c523)

Note also that the full.dribble file is garbage.

**To Reproduce**
1. Pick a file that is loaded during the loadup-full process, add (P (ERROR)) to its commands, run loadup-full.sh.

A too-small and incomplete full.sysout shows up in the loadups directories.  The shell script reports SUCCESS

**Expected behavior**
The shell script reports FAILURE, the sysout file doesn't show up in the loadups directory.
It would be nice, to help in debugging the source of the error, if the dribble file was intact.

**Screenshots**

**Context (please complete the following information):**
 - OS: Mac
 - OS Version:  Sonoma 14.1
 - Host arch: arm64,

**Comment by rmkaplan** [comment]
Is there a function that can be called from Lisp that takes it into URAID (perhaps with a message)?

If so, then perhaps we can signal failure by wrapping the various loadup functions under NLSETQs, and dropping into URAID on error.  And then, maybe doing URAID's hard-reset or quit commands would get back to a debuggable state inside the sysout.

**Comment by nbriggs** [comment]
>Is there a function that can be called from Lisp that takes it into URAID (perhaps with a message)?

`(RAID "message1")`

**Comment by masinter** [comment]
normally if a loadup fails, it doesn't thwn go on and do the MAKESYS.
or make a new whereis.hash, etc

that was the reason for the 'toudh' of the timestamp file. Failure can happen for too many unpreductavke reasona to guard against it with a NLSETQ.

**Comment by rmkaplan** [comment]
That's not true:  if you put an ERROR explicitly in the loadup function, the MAKESYS gets made and moved to loadups

**Comment by masinter** [comment]
I think if an error happens during LOADUP-FULL it should error, enter a break.
set NLSETQGAG to NIL.
